tests/test_helpers.py

```diff
@@ -941,3 +942,38 @@ def test_safe_join_exceptions(self):
         for args in failing:
             with pytest.raises(NotFound):
                 print(flask.safe_join(*args))
+
+class TestHelpers(object):
+
+    def test_get_debug_flag(self, monkeypatch):
+        monkeypatch.setenv('FLASK_DEBUG', '')
+        assert get_debug_flag() == None
+        assert get_debug_flag(default=True) == True
```

Comment:
I think you can omit the `== True`? Since the assert automatically checks if the statement evaluates to `True`... This is true for a bunch of tests below as well. 

Similarly you could rewrite:

    assert FalseThing == False

as:

    assert not FalseThing

I find the latter cleaner, but I guess this is a matter of personal preference as it's arguably less explicit.